src/components/SideMenu/SideMenu.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+
+import SideMenuStyle from './SideMenuStyle';
+import SideMenuItem from 'Components/SideMenuItem/';
+
+const displayName = 'SideMenu';
+
+const SideMenu = ({ sports, sportItemClick, currentSportId }) => {
+  return (
+    <SideMenuStyle>
+      {sports
+        ? sports.entrySeq().map(([key, val]) => (
```

Comment:
As I remember `Map` perfectly iterated without `entrySeq`.
```javascript
sports.map((sport, key) =>
```

Reply:
It is done to use map() of vanilla JS, because React won't support map of immutable.js

Reply:
Ok, agree